progress/upload_status.py

```diff
@@ -78,14 +93,19 @@ def write_directory_status(directory_status, run_id=None):
     if not os.access(directory_status.directory, os.W_OK):  # Cannot access upload directory
         raise exceptions.DirectoryError("Cannot access directory", directory_status.directory)
 
+    # if directory_status.is_invalid():
```

Comment:
Remove this

Reply:
Fixed in effa23ba966ebf6801960a9a3075812ff0446a91